### DISCUSSION TITLE
Camera position and noise

### DISCUSSION THREAD

**Discussion by amagasso** [empriselab/RCareWorld#154] [Other Q&A] [answered]
Hi organizers,
In the template bathing scene, it is mentioned that we can add as many cameras as we want (less than 5 for smooth simulations).
1. Are we allow to position them as we want (e.g., floating somewhere) although it is not realistic in real-world settings?
2. In the evaluation, is there any sensing noise from cameras (e.g. depth?)

**Comment by YoruCathy** [comment, marked as answer]
Yes, feel free to position them anywhere. There weren't be any noise. We do have the depth with noise mimicking the real world ones but you don't have to use it.

**Reply by amagasso** [reply]
Thank you for the quick answer
